cargo-miri/bin.rs

```diff
@@ -461,7 +461,10 @@ path = "lib.rs"
     command.env_remove("RUSTFLAGS");
     // Disable debug assertions in the standard library -- Miri is already slow enough.
     // But keep the overflow checks, they are cheap.
-    command.env("RUSTFLAGS", "-Cdebug-assertions=off -Coverflow-checks=on");
+    command.env(
+        "RUSTFLAGS",
+        "-Cdebug-assertions=off -Coverflow-checks=on -Zremap-cwd-prefix=rustc_src",
```

Comment:
Does the regular std build also do this?
Either way, please extend the comment above to explain this flag.

Reply:
Well, no, but the regular rust build doesn't rebuild libstd on every machine 😄

This flag exists for reproducible builds and is used by linux distros